ros/models.py

```diff
@@ -6,12 +6,15 @@
 
 
 class PerformanceProfile(db.Model):
-    __table_args__ = (db.UniqueConstraint('inventory_id', 'report_date'), )
     id = db.Column(db.Integer, primary_key=True)
-    inventory_id = db.Column(UUID(as_uuid=True), nullable=False)
     performance_record = db.Column(JSONB)
     performance_score = db.Column(JSONB)
     report_date = db.Column(db.Date, default=date.today())
+    system_id = db.Column(db.Integer, primary_key=True)
+    __table_args__ = (
+        db.UniqueConstraint('report_date'),
```

Comment:
More than one system can report on same day then will it work by declaring unique_key only on `report_date` column? 
It should consider single unique_key using both `report_date` & `system_id`columns.

Reply:
Oh I must've missed that. The primary key should be `(system_id,report_date)`

Reply:
Yeah, @apuntamb is working on changing this table to use `system_id, report_date` as a primary key.